hack/community/generate.sh

```diff
@@ -0,0 +1,113 @@
+      .metadata.annotations.createdAt |= strenv(CREATED_AT) |
+      .metadata.name |= strenv(NAME) |
+      .spec.description |= strenv(CO_DESCRIPTION) |
+      .spec.displayName |= strenv(DISPLAY_NAME) |
+      .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[2].value |= strenv(VALUE) |
+      .spec.install.spec.deployments[0].spec.template.spec.containers[0].image |= strenv(QUAY_IMAGE) |
+      .spec.maturity |= strenv(MATURITY) |
+      .spec.version |= strenv(VERSION)
+    ' "${CO_CSV}"
+}
+
+# copy the WMCO bundle and its contents to a new community folder. Rename files
+# as needed.
+generate_manifests() {
+  local CO_OPERATOR_DIR="${CO_DIR}/operators/community-windows-machine-config-operator"
+
+  echo "Update operator manifests"
+  mkdir "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}"
+  cp -r "${OPERATOR_MANIFESTS}" "${OPERATOR_METADATA}" "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}"
+  mv "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/manifests/windows-machine-config-operator.clusterserviceversion.yaml" "${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/manifests/community-windows-machine-config-operator.${OPERATOR_VERSION}.clusterserviceversion.yaml"
+  pushd "${CO_DIR}"
+  git checkout -B "${OPERATOR_VERSION}"
+  CO_CSV="${CO_OPERATOR_DIR}/${OPERATOR_VERSION}/manifests/community-windows-machine-config-operator.${OPERATOR_VERSION}.clusterserviceversion.yaml"
+
+  replace "$CO_CSV" "$OPERATOR_VERSION" "$DESCRIPTION" "$QUAY_IMAGE"
+}
+
+# create a signed commit for the community-operators-prod repo
+commit() {
+  TITLE="operators community-windows-machine-config-operator.v${OPERATOR_VERSION}"
+  MESSAGE="Signed of by: ${GITHUB_NAME} ${GITHUB_EMAIL}"
+
+  echo "commit changes"
+  git add --all
+  git commit -s -m "${TITLE}
+  ${MESSAGE}"
+  popd
+}
+
+OPERATOR_VERSION="$1"
+COMMUNITY_VERSION="$2"
+QUAY_IMAGE=REPLACE_IMAGE
+OPERATOR_MANIFESTS="bundle/manifests"
+OPERATOR_METADATA="bundle/metadata"
+CO_PROJECT="community-operators-prod"
+CO_REPO="redhat-openshift-ecosystem/${CO_PROJECT}"
+CO_FORK="https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${GITHUB_USER}/${CO_PROJECT}.git"
+CO_DIR=$(mktemp -d)
+
+# Check if yq is installed
+if ! command -v which yq &> /dev/null; then
+  echo "Please install yq before running generate.sh. Directions to install yq are in the readme.md"
+  exit 1
+fi
+
+# check if ENV vars are set
+if [ -z "${GITHUB_USER}" ] || [ -z "${GITHUB_TOKEN}" ] || [ -z "${GITHUB_NAME}" ] || [ -z "${GITHUB_EMAIL}" ]; then
+    echo "Must set all of GITHUB_USER, GITHUB_TOKEN, GITHUB_NAME, and GITHUB_EMAIL"
+    exit 1
+fi
+
+# check if QUAY_IMAGE is replaced
+if [ "$QUAY_IMAGE" == "REPLACE_IMAGE" ]; then
+    echo "Must set QUAY_IMAGE to latest community tag"
+    exit 1
+fi
+
+echo "Cutting community release for v${OPERATOR_VERSION}"
+
+# Inject appropriate community-version into the description
+sed -i "s/COMMUNITY_VERSION/$COMMUNITY_VERSION/" hack/community/csv/description.md
+DESCRIPTION=$(cat hack/community/csv/description.md)
```

Comment:
This is making a local change to the non-cloned repo first. To avoid this you consider:
```suggestion
DESCRIPTION=$(cat hack/community/csv/description.md)
DESCRIPTION=${DESCRIPTION//COMMUNITY_VERSION/$COMMUNITY_VERSION}
```

Reply:
Thanks, added this in